.github/workflows/e2e-infra-cleanup.yaml

```diff
@@ -0,0 +1,78 @@
+name: e2e-infra-cleanup
+
+on:
+  workflow_dispatch:
+    inputs:
+      cleanup_interval_hours:
+        description: Cleanup interval in hours
+        required: true
+        default: "24"
+    secrets:
+      E2E_TESTIM_AWS_ACCESS_KEY_ID:
+        required: true
+      E2E_TESTIM_AWS_SECRET_ACCESS_KEY:
+        required: true
+      E2E_GH_PAT:
+        required: true
+  schedule:
+    - cron: "0 * * * *"
+
+concurrency: e2e-infra-cleanup
+
+env:
+  AWS_DEFAULT_REGION: us-east-1
+  AWS_ACCESS_KEY_ID: ${{ secrets.E2E_TESTIM_AWS_ACCESS_KEY_ID }}
+  AWS_SECRET_ACCESS_KEY: ${{ secrets.E2E_TESTIM_AWS_SECRET_ACCESS_KEY }}
+  CLEANUP_INTERVAL_HOURS: ${{ github.event.inputs.cleanup_interval_hours || '24' }}
+
+jobs:
+  find-expired-workspaces:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: automation/jumpbox
+    outputs:
+      workspaces: ${{ steps.get-workspaces.outputs.workspaces }}
+      count: ${{ steps.get-workspaces.outputs.count }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          repository: replicatedhq/kots-regression-automation
+          token: ${{ secrets.E2E_GH_PAT }}
+          path: automation
+          ref: main
+      - name: Setup Terraform
+        uses: hashicorp/setup-terraform@v2
+        with:
+          terraform_wrapper: false
+      - name: Initialize terraform
+        run: terraform init -reconfigure
+      - name: Get automation workspaces
+        id: get-workspaces
+        run: |
+          WORKSPACES=($(terraform workspace list | grep "nickm"))
```

Comment:
Remove the grep?

Reply:
nope, but the value's wrong. Nice catch.